### PR TITLE
Add filter chips

### PR DESCRIPTION
**Pull request by LanesGood**
Adds filter chips.

### CODE REVIEW

**Comment by vgeorge** [comment]
@LanesGood comments:

- What if each filter type occupied a paragraph/line? It looks a little bit confusing when multiple are selected
- It seems that the spacing between filter type and selected filter chips are equal, can we reduce the spacing between the selected ones?

Screenshot 2020-10-20 at 10 39 14

- Org sizes chips should include `>=`

**Comment by LanesGood** [comment]
@vgeorge we can definitely have each filter type on its own line - I was trying to save space, but can instead reduce some of the section spacing. Same for same-filter-type button spacing.

For org size chips and remote options, the filter is displaying the filter `id` rather than the `label`. I'm working on getting the label to display instead.

**Comment by LanesGood** [comment]
@vgeorge I've updated spacing. I don't think the filter `label` is easily accessible in the current set up. @necoline do you have any quick tips for including the `label` instead of the `id` for the filter value? If this requires an overly complicated setup, I would suggest that it is acceptable as is.

File: src/components/filters.js [hunk 1]
```diff
@@ -91,6 +136,7 @@ const initialFilterState = {
 function Filters({ fullGuide, setGuide, activities }) {
   const [selectedFilters, setFilter] = useState(initialFilterState)
   const [allActivities, setAllActivities] = useState([])
+  console.log(selectedFilters)
```
Comment (necoline) [inline src/components/filters.js:139]: extra console

**Review by necoline — APPROVED** [review]
I don't think it is worth reorganizing the filtering logic to include the label for the chips, but a few other options could be
1) raising the option arrays to the component level rather than in the jsx and then in the `filterChips` use the `filterKeys` to reference the option id or label if a label exists

2) if `filterKey` is `orgSize` add <= to the value and leave the remote options as is

**Comment by LanesGood** [comment]
@necoline thanks for the suggestions - I decided to go with adding the `<=` symbol for the `orgSize` filter only. 

Ready for re-review.

**Pull request merged**